libcontainer/configs/config.go

```diff
@@ -72,8 +72,12 @@ type Config struct {
 	// bind mounts are writtable.
 	Readonlyfs bool `json:"readonlyfs"`
 
-	// Privatefs will mount the container's rootfs as private where mount points from the parent will not propogate
-	Privatefs bool `json:"privatefs"`
+	// RootfsMountMode is the rootfs mount propagation mode.
+	// On linux it is one of the followings:
+	// "private": rootfs is mounted as MS_PRIVATE
+	// "shared": rootfs is mounted as MS_SHARED
+	// "slave": rootfs is mounted as MS_SLAVE
+	RootfsMountMode PropagationMode `json:"root_mount"`
```

Comment:
Shouldn't this be something like `rootmountmode` to fit the pattern of the other fields' serialized representations?